experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import { Container } from "@fluidframework/container-loader";
+import { ICriticalContainerError } from "@fluidframework/container-definitions";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { IEvent } from "@fluidframework/common-definitions";
+import { LoadableObjectClass, LoadableObjectRecord } from "./types";
+import { RootDataObject } from "./rootDataObject";
+interface IFluidContainerEvents extends IEvent {
+    (event: "connected", listener: (clientId: string) => void): void;
```

Comment:
Just thinking out loud, no changes requested right now for this - maybe we should not include the clientId here?  No strong opinion yet but I'm thinking if this interface is to be used by a host, is the clientId appropriate for them to be leveraging or should it be considered an internal/implementation detail?  And/or if they did want it, is there a better access point?

Reply:
since we're moving get clientId onto the service, it'd make sense to remove it from this event as well